build_tools/cmake/build_all.sh

```diff
@@ -47,6 +48,7 @@ declare -a CMAKE_ARGS=(
 
   # Enable building the python bindings on CI.
   "-DIREE_BUILD_PYTHON_BINDINGS=ON"
+  "-DPython3_EXECUTABLE=${IREE_PYTHON3_EXECUTABLE}"
```

Comment:
I think you also want to set `-DPYTHON_EXECUTABLE`. At least I've got that in some of my CMake settings. Maybe even the include dirs? https://github.com/iree-org/iree/blob/c1a2645da619fe517376617fcd3ec94c2624fd27/build_tools/github_actions/cmake_ci.py#L152

Reply:
This is the only variable mentioned in our docs (pretty sure Stella wrote these after similar troubleshooting): https://iree-org.github.io/iree/building-from-source/python-bindings-and-importers/#:~:text=build_requirements.txt.-,CMake%20Variables,-%3A

Reply:
(and it seems to work, let's keep it simple if possible :P)

Reply:
SG. I think this maybe improved with later CMake versions